test/amd.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>QUnit AMD Test Suite</title>
+	<link rel="stylesheet" href="../dist/qunit.css">
+</head>
+<body>
+	<div id="qunit"></div>
+	<script>
+	var beginData,
+		asyncDelay = 1000,
+		times = {};
+
+	(function() {
+		var now = Date.now || function() {
+			return new Date().getTime();
+		};
+
+		// Simulate a delay in loading the tests, as when they are loaded
+		// asynchronously using requirejs, steal, etc.
+		window.define = function() {};
+		window.define.amd = true;
+		setTimeout(function() {
+			var script = document.createElement( "script" );
+			script.src = "../dist/qunit.js";
+			script.onload = function loadTests() {
+				QUnit.begin(function( data ) {
+					beginData = data;
+					times.runStarted = now();
+				});
+				var script = document.createElement( "script" );
+				script.src = "amd.js";
+				document.getElementsByTagName( "head" )[ 0 ].appendChild( script );
+			};
+			times.pageLoaded = now();
```

Comment:
I'm sorry going so deep here, but I believe this pageLoaded < runStarted won't assert anything, pageLoaded will always be lower than runStarted in this case.

At this point, it's good to get rid of the times object and the respective assertion. The beginData assertion will be enough until we find a better approach.